### PR TITLE
use signed JWT tokens instead of the whole of token-service

### PR DESCRIPTION
**Pull request by willp-bl**
* this uses test-rp's primary signing key to sign tokens
* there's a ruby script to generate valid tokens
* to force revoke all currently issued tokens increase the epoch by one

things to consider that I won't make changes for in this PR:

- [ ] is this using the correct signing key?
- [ ] how long to generate tokens for? (see https://github.com/alphagov/verify-architecture/pull/58#issuecomment-443167779)
- [ ] what are the values that should go in the issued_to field
- [ ] where does the script to generate tokens live?
- [ ] has use of this been documented in the team manual etc

ps when I start to get spammed by the bot on slack about this PR I may or may not close it

### CODE REVIEW

File: tokens/generate_token.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+require 'openssl'
+
+rsa_private = OpenSSL::PKey::RSA.new File.read 'dev-keys/sample_rp_signing_primary.private.pem'
+rsa_public = rsa_private.public_key
+
+payload = { epoch: 1,
+            valid_until: '2118-11-27T11:30:00.000Z',
+            issued_to: 'willp-bl' }
+
+payload = {}
```
Comment (jakubmiarka) [inline tokens/generate_token.rb]: why is `payload` initiated with values and then set to an empty object?

Comment (willp-bl) [reply tokens/generate_token.rb]: I left this in accidentally, but it's just a proof of concept script :)

Comment (willp-bl) [reply tokens/generate_token.rb]: used this to generate the tokens that are used in the integration tests

**Comment by stephengrier** [comment]
This all looks good to me. There is very little risk in merging this because we don't appear to be using the token service in joint so this won't break the acceptance tests, and AFAIK there isn't anyone using tokens to access test-rp in prod at the moment, and the headless stub isn't restricted by tokens anyway. So big 👍 from me.

**Comment by tlwr** [comment]
Can confirm that token service is not being used in staging anymore

The only users (if any) will be external

**Comment by willp-bl** [comment]
See #27

**Pull request closed without merging**